.copr/yggdrasil.spec.in

```diff
@@ -0,0 +1,80 @@
+%bcond_without check
+
+# https://github.com/redhatinsights/yggdrasil
+%global goipath         github.com/redhatinsights/yggdrasil
+%global commit          @COMMIT@
+
+%gometa -f -i
+
+# Manually redefine %%dist to work around an issue in COPR where the build root
+# that creates the srpm does not define a value for %%dist. This should *NOT* be
+# carried in downstream; this is strictly an upstream/COPR/CI workaround.
+%global dist %{distprefix}.fc%{fedora}
+
+%global common_description %{expand:
+yggdrasil is a system daemon that subscribes to topics on an MQTT broker and
+routes any data received on the topics to an appropriate child "worker" process,
+exchanging data with its worker processes through a D-Bus message broker.}
+
+%global golicenses      LICENSE
+%global godocs          CONTRIBUTING.md README.md
+
+Name:           yggdrasil
+Version:        0.3.1
+Release:        %autorelease -p
+Summary:        Remote data transmission and processing client
+
+License:        GPL-3.0-only
+URL:            %{gourl}
+Source:         %{gosource}
+
+BuildRequires:  systemd-rpm-macros
+BuildRequires:  meson
+BuildRequires:  pkgconfig(dbus-1)
+BuildRequires:  pkgconfig(systemd)
+BuildRequires:  pkgconfig(bash-completion)
+
+%description %{common_description}
+
+%gopkg
+
+%prep
+%goprep
```

Comment:
Please add again comment with brief description to `%goprep`. (something like: "setup -q" for RPM package implemented in Go)

Reply:
I don't think it's necessary to document the standard Go packaging macros. Their meaning and function are second nature to any packager used to the Fedora Go package tooling. If it's unfamiliar to you now, that's only because it's new and you have not worked with it for very long yet. And again, I'd rather avoid carrying unrelated to yggdrasil that's better suited in the [Fedora Packaging Guidelines](https://docs.fedoraproject.org/en-US/packaging-guidelines/Golang/#_prep_goprep_removing_vendoring).

Reply:
OK